tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Debug/DebugTokenReplacerTests.cs

```diff
@@ -10,23 +10,27 @@
 
 namespace Microsoft.VisualStudio.ProjectSystem.Debug
 {
-
     public class DebugTokenReplacerTests
     {
-        private readonly Dictionary<string, string> _envVars = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase) {
+        private readonly Dictionary<string, string> _envVars = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+        {
             { "%env1%","envVariable1" },
             { "%env2%","envVariable2" },
             { "%env3%","$(msbuildProperty6)" }
         };
-        private readonly Mock<IEnvironmentHelper> _envHelper = new Mock<IEnvironmentHelper>();
+
+        private readonly Mock<IEnvironmentHelper> _envHelper;
+        
         public DebugTokenReplacerTests()
         {
+            _envHelper = new Mock<IEnvironmentHelper>();
```

Comment:
Just to confirm I'm not missing something, this is just stylistic right?

Reply:
Yes, stylistic. I like construction and initialization together.